spec/support/methods/feature_test_helper_methods.rb

```diff
@@ -0,0 +1,45 @@
+module FeatureTestHelperMethods
+  def start(the_flow:, at:)
+    visit "/#{at}"
+    ensure_page_has(header: the_flow)
+    click_govuk_start_button
+  end
+
+  def answer(question:, of_type:, with:)
+    ensure_page_has(header: question)
+    case of_type
+    when :radio_button
+      choose with
+    when :check_box
+      check with
+    when :check_boxes
+      raise "For the `check_boxes` type, `with` should be an array" unless with.is_a?(Array)
+
+      with.each do |answer|
+        check answer
+      end
+    when :input_text
+      fill_in "response", with: with
+    when :date
+      raise "For the `date` type, `with` should be a hash" unless with.is_a?(Hash)
+
+      fill_in "response[day]", with: with[:day]
+      fill_in "response[month]", with: with[:month]
+      fill_in "response[year]", with: with[:year]
+    end
+    click_button "Continue"
+  end
+
+  def ensure_page_has(header: nil, subheader: nil, subheaders: [], text: nil)
```

Comment:
So I'm wondering if we need to the subheaders? Its only really the find-support flow that has them I think?

But if so you might be able to roll subheaders and subheaders in to a single argument - the expectation is somewhat the same.
```suggestion
  def ensure_page_has(header: nil, subheaders: [], text: nil)
```

Reply:
I'd prefer to keep `subheaders` as they are used in the find-support flow and testing the `h1` there doesn't make sense or help much as it's always the same!  However, I've removed `subheader` and updated the code so that it will convert to Array - if a String is given.  This way at least there is only one optional param for `subheaders` now.